sds_data_manager/stacks/indexer_lambda_stack.py

```diff
@@ -76,3 +80,92 @@ def __init__(
             self, "rds_secret", db_secret_name
         )
         rds_secret.grant_read(grantee=indexer_lambda)
+
+        # Instrument list
+        instruments = [
+            "codice",
+            "glows",
+            "hi",
+            "hit",
+            "idex",
+            "lo",
+            "mag",
+            "swapi",
+            "swe",
+            "ultra",
+        ]
+
+        # Events that triggers Indexer Lambda:
+        # 1. Arrival of L0 data
+        # 2. PutEvent from Lambda that builds dependency and starts Batch Job
+        # 3. Batch Job status change
+
+        # Write l0 info to db with
+        # status SUCCEEDED
+        l0_arrival_rule = events.Rule(
+            self,
+            "l0DataArrival",
+            rule_name="l0-data-arrival",
+            event_pattern=events.EventPattern(
+                source=["aws.s3"],
+                detail_type=["Object Created"],
+                detail={
+                    "bucket": {"name": [data_bucket.bucket_name]},
+                    "object": {
+                        "key": [
+                            {"prefix": f"imap/{instrument}/l0/"}
```

Comment:
Not for this PR, but just a note for discussion in our next tag-up.

I was just chatting with @sdhoyt this morning about whether we have `/<mission>/` as an entry in the DB to search by anymore. I know we talked in the past about getting I&T data and running that through our pipeline. I'm wondering if we should add a new qualifier to our bucket to put things in a different base "mission" area, but still run everything the same if we can.